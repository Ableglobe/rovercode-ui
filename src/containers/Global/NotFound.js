import React from 'react';
import { Grid, Header, Image } from 'semantic-ui-react';

import notFoundImage from '@/assets/images/404.png';

const NotFound = () => (
  <Grid centered>
    <Grid.Row>
      <Image src={notFoundImage} />
    </Grid.Row>
    <Grid.Row>
      <Header size="huge">
        Sorry, the page you&apos;re looking for doesn&apos;t exist.
      </Header>
    </Grid.Row>
  </Grid>
);

export default NotFound;
