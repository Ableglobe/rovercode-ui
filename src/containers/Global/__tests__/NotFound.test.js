import React from 'react';
import { shallow } from 'enzyme';

import NotFound from '../NotFound';

test('NotFound renders on the page with no errors', () => {
  const wrapper = shallow(<NotFound />);

  expect(wrapper).toMatchSnapshot();
});
