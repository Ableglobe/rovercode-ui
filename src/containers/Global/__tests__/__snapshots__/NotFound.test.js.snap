// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound renders on the page with no errors 1`] = `
<Grid
  centered={true}
>
  <GridRow>
    <Image
      as="img"
      src={Object {}}
      ui={true}
    />
  </GridRow>
  <GridRow>
    <Header
      size="huge"
    >
      Sorry, the page you're looking for doesn't exist.
    </Header>
  </GridRow>
</Grid>
`;
